### PR TITLE
Fix nvcc warnings in make all [compiler-warnings]

### PR DESCRIPTION
**Pull request by camierjs**
~I left in the first commit the compiler complaints. The authors can get rid of these lines if wanted.~
I removed the lines.

### CODE REVIEW

File: miniapps/electromagnetics/joule.cpp [hunk 1]
```diff
@@ -115,14 +115,14 @@ using namespace mfem::electromagnetics;
 
 void display_banner(ostream & os);
 
-static double aj_ = 0.0;
+//static double aj_ = 0.0; // set but never used
```
Comment (tzanio) [inline miniapps/electromagnetics/joule.cpp]: @mlstowell, can you take a look at these?

File: tests/unit/fem/test_2d_bilininteg.cpp [hunk 2]
```diff
@@ -2090,7 +2090,7 @@ TEST_CASE("2D Bilinear Scalar Weak Curl Integrators",
           "[NonlinearFormIntegrator]")
 {
    int order = 2, n = 1, dim = 2;
-   double cg_rtol = 1e-14;
+   // double cg_rtol = 1e-14; //declared but never referenced
```
Comment (tzanio) [inline tests/unit/fem/test_2d_bilininteg.cpp]: I think these are fine to remove.

Comment (acfisher) [reply tests/unit/fem/test_2d_bilininteg.cpp]: Agreed.  No compelling reason to leave them in and they pollute our compiler warnings.

Comment (camierjs) [reply tests/unit/fem/test_2d_bilininteg.cpp]: Done

**Comment by tzanio** [comment]
@Dan2997925, we merged this for testing, but we can still revert the variables in Joule if you let us know early next week (the mfem-4.0 release is scheduled for Wednesday)

**Pull request merged**